velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -15,7 +15,12 @@
  */
 #include "velox/common/file/FileSystems.h"
 
+namespace velox::filesystems::arrow::io::internal {
```

Comment:
Is this change necessary?  I can see in `HdfsFileSystem.cpp`, it includes `ArrowHdfsInternal.h`, which has `LibHdfsShim` already.

Btw, is the namespace correct here? Should be `facebook::velox::filesystems::arrow::io::internal`??